openmetadata-ui/src/main/resources/ui/src/components/Settings/Team/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -267,10 +268,38 @@ const TeamDetailsV1 = ({
 
   const searchTeams = async (text: string) => {
     try {
-      const res = await getSearchedTeams(text, 1, '');
```

Comment:
I think we can remove this method if no longer in use?

Reply:
done, removed!